### ISSUE TITLE
Create proper README and other project documentation

### ISSUE THREAD

**Issue by eAlasdair** [uccser/codewof#258]
Self explanatory

**Comment by CyberFlameGO** [comment]
Oh, like from [uccer/cs-unplugged /docs/source](https://github.com/uccser/cs-unplugged/tree/develop/docs/source)?

**Comment by JackMorganNZ** [comment]
@CyberFlameGO Correct. Similar structure and a lot of the content will be the same due to the similarity in the systems. This sadly just isn't high on our prority, especially as I'm the only dedicated technical developer currently.
